website/docs/api/plugins/overview.md

```diff
@@ -27,3 +27,9 @@ These plugins will add a useful behavior to your Docusaurus site.
 - [@docusaurus/plugin-ideal-image](./plugin-ideal-image.md)
 - [@docusaurus/plugin-google-analytics](./plugin-google-analytics.md)
 - [@docusaurus/plugin-google-gtag](./plugin-google-gtag.md)
+
+## Development plugins {#dev-plugins}
+
+These plugins are useful for developing a site with Docusaurus.
+
+- [@docusaurus/eslint-plugin](./eslint-plugin.md)
```

Comment:
Ditto—remove this from plugins/overview because it's not a Docusaurus plugin and may confuse users

Reply:
Sure - 9f04ae1.